client/js/components/snippets/snippets-list.jsx

```diff
@@ -1,7 +1,20 @@
 import React from 'react';
 import Snippet from './snippet';
+import CommentBox from '../comments/comment-box';
 
-export default class SnippetsIndex extends React.Component {
+class SnippetListItem extends React.Component{
+  render() {
+    return (
+      <li style={{position: 'relative', paddingLeft: '30px'}} key={this.props.snippet.id}>
+        <span style={{position: 'absolute', left: 0}}>#{this.props.index+1}</span>
+        <Snippet {...this.props.snippet} style={{marginBottom: '15px'}} />
+        <CommentBox snippetId={this.props.snippet.id} comments={this.props.snippet.comments}/>
```

Comment:
@vrael560 I'm just wondering, wouldn't be easier if ```CommentBox``` was a part of Snippet component?